### PR TITLE
Simorgh - Storybook Upgrade

### PR DESCRIPTION
**Pull request by ryanmccombe**
Part of https://github.com/bbc/simorgh/issues/9198

**Overall change:**
_Upgrade to latest version of Storybook._

**Code changes:**

- _Add the a11y plugin and viewport plugin to package.json - we seem to be trying to use these in the storybook config, but they were never installed, so this never worked.  This PR now correctly shows the "A11y" tab in storybook._
- _Install addon-controls - the replacement for addon-knobs._
- _Install addon-docs - we appear to be trying to use addon-notes in some stories to populate the docs tab with the contents of a markdown file - eg https://github.com/bbc/simorgh/blob/935c898e3bf8ae568ae89620b7461cdb768853ae/src/app/components/MediaPlayer/index.stories.jsx#L10, but the plugin is never installed or added to the config, so this never worked.  add-on notes is now stale - addon-docs seems preferred

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -171,7 +166,11 @@
     "@emotion/jest": "11.3.0",
     "@loadable/babel-plugin": "5.13.2",
     "@loadable/webpack-plugin": "5.15.0",
+    "@storybook/addon-a11y": "6.3.1",
+    "@storybook/addon-controls": "6.3.1",
+    "@storybook/addon-docs": "6.3.1",
     "@storybook/addon-knobs": "6.2.9",
+    "@storybook/addon-viewport": "6.3.1",
     "@storybook/react": "6.2.9",
```
Comment (HarveyPeachey) [inline package.json]: Does this need to be changed to `6.3.1`? When I checkout the branch and run `yarn` and then `yarn run storybook`, I get a notification in the terminal saying `A new version (6.3.1) is available!`

**Review by HarveyPeachey — COMMENTED** [review]
Nice catch on the plugin fixes!

**Review by HarryVerhoef — APPROVED** [review]
Nice one :rocket: I think they've just released 6.3.2 so may be worth bumping again, but I'm sure 6.3.1 is fine too

File: package.json [hunk 2]
```diff
@@ -8,17 +8,12 @@
...
     "@emotion/utils": "1.0.0",
     "moment": "2.29.1",
-    "ssri": "8.0.1",
-    "hosted-git-info": "3.0.8",
-    "postcss": "8.3.0",
     "browserslist": "4.16.5",
-    "multicast-dns": "7.2.3",
     "web-vitals": "1.1.1",
-    "ws": "7.4.6",
-    "trim-newlines": "3.0.1",
```
Comment (andrewscfc) [inline package.json:18]: I guess these are the various pinned dependencies we had to avoid security issues that are no longer needed?

**Review by andrewscfc — APPROVED** [review]
Thanks for doing this upgrade 👍 

So to make sure I understand we updating to the latest stable version of Storybook 6 to solve security problems?

I thought we were going to upgrade to a Storybook 7 rc to get around the emotion support problems too? Looks like that isn't finished though judging by this issue: https://github.com/storybookjs/storybook/issues/13491

**Comment by ryanmccombe** [comment]
> So to make sure I understand we updating to the latest stable version of Storybook 6 to solve security problems?

Storybook 6.3 was the RC version at the time we had that call - it is now stable.  I don't think a 7.0 RC is available yet, unless I'm missing something - https://www.npmjs.com/package/@storybook/react

File: package.json (hunk 2 above)
Comment (ryanmccombe) [reply package.json:18]: Yep :)

**Comment by ryanmccombe** [comment]
This does seem to have caused 2 stories to stop working - the next PR is a rewrite of all stories, however, so will see if this is still an issue then.  Confirmed the components still work, so the problem is restricted to storybook

**Pull request merged**